### ISSUE TITLE
Vuetify needs updating

### ISSUE THREAD

**Issue by apap04** [nuxt/nuxt.js#6337]
### What problem does this feature solve?

Vuetify needs to be updated to the latest version. When I try to manually update vuetify, I get a bunch of errors when I try to build the project.

### What does the proposed changes look like?

The Vuetify example should be updated so users can use the latest features of Vuetify.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9713">#c9713</a>)</em></sub></div>

**Comment by manniL** [comment]
I agree that the example should be updated and https://github.com/nuxt-community/vuetify-module/ should be used ☺️
